<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sportsed::Client::ConnectionDialog</class>
 <widget class="QDialog" name="Sportsed::Client::ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to server</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <widget class="QStackedWidget" name="settingsStackedWidget">
     <widget class="QWidget" name="connectionOptionsWidget" native="true">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Host:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="hostEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="passwordEdit">
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="competitionSelectionWidget">
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Competition:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="competitionsBox"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="refreshCompetitionsButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QPushButton" name="createCompetitionBtn">
         <property name="text">
          <string>Create new competition</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLbl">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
